### DISCUSSION TITLE
How to use configuration variables?

### DISCUSSION THREAD

**Discussion by MattBlanco** [nektos/act#1680] [Q&A] [answered]
So github added [configuration variables](https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/) a few months ago that you refer to using `vars.VARIABLE_NAME` does act support them already?

**Comment by KnisterPeter** [comment, marked as answer]
No support yet.
Feel free to provide a PR and contribute that feature.
